Setup/EavOptionSetup.php

```diff
@@ -50,18 +51,26 @@ class EavOptionSetup
      */
     private $appState;
 
+    /**
+     * @var ResourceConnection
+     */
+    private $resurceConnection;
+
     public function __construct(
         AttributeRepository $attributeRepository,
         AttributeOptionManagementService $attributeOptionManagementService,
         AttributeOptionFactory $attributeOptionFactory,
         AttributeOptionLabelFactory $attributeOptionLabelFactory,
-        AppState $appState
-    ) {
+        AppState $appState,
+        ResourceConnection $resurceConnection
```

Comment:
Typo: `$resurceConnection` is missing an `o` in `resource` (same as above, just for the constructor argument variable name).